configs/testing/exp_eufy_t8441

```diff
@@ -0,0 +1,6 @@
+# MODULE: t31x_sc3335_syn43436_defconfig
+FLASH_SIZE_32=y
+U_BOOT_ENV_TXT="$(BR2_EXTERNAL)/environment/eufy-t8441.uenv.txt"
+BR2_AUDIO=y
+BR2_PACKAGE_SOX=y
+BR2_PACKAGE_EXFAT_NOFUSE=y
```

Comment:
instead of manually adding exfat nofuse, use `BR2_SDCARD=y` which will select it.